data-raw/northern-ireland-hsct-secondary-care.R

```diff
@@ -0,0 +1,147 @@
+# Northern Ireland - Secondary Care Indicators
+# Health and Social Care Trusts
+
+# ---- Load libs & helpers ----
+library(tidyverse)
+library(geographr)
+library(sf)
+library(healthyr)
+library(demographr)
+library(compositr)
+library(ggridges)
+library(devtools)
+library(readODS)
+
+hsct <- boundaries_trusts_ni18 |>
+  st_drop_geometry()
+
+lookup_trusts <- geographr::boundaries_trusts_ni18 |>
+  st_drop_geometry()
+
+# Population counts
+# Source: https://www.ninis2.nisra.gov.uk/public/AreaProfileReportViewer.aspx?FromAPAddressMulipleRecords=Northern@@Northern@20?
+
+url <- "https://www.ninis2.nisra.gov.uk/Download/Population/Population%20Estimates%20Broad%20Age%20Bands%20(administrative%20geographies).ods"
+
+pop_raw <- download_file(url, ".ods")
+
+pop_ni_hsct <- read_ods(
+  pop_raw,
+  sheet = "HSCT",
+  range = "A4:C9"
+) |>
+  select(
+    trust18_name = "HSCT",
+    trust18_code = "HSCT Code",
+    population = "Persons"
+  )
+
+# ---- RTT ----
+# Higher = worse performance
+rtt <- ni_rtt_hsct |>
+  select(
+    trust18_name = "hsct22_name",
+    date,
+    waits_over_18_weeks
+  ) |>
+  filter(date >= max(date) %m-% months(2)) |> # Last quarter
```

Comment:
`filter(date >= max(date) %m-% months(2))` doesn't appear to be working as it is returning data from 2019:

<img width="569" alt="Screenshot 2023-07-13 at 17 23 42" src="https://github.com/britishredcrosssociety/health-inequalities-explorer/assets/38110953/11f88c9b-f9c2-443f-b08a-8bd067840626">

Please can you look into this more and alter as required to return the last three months of data?

Reply:
I have managed to solve this using ungroup() because the ni_rtt_hsct dataset was grouped which is why it would add the "year" and "month" columns. Would you like me to ungroup the dataset in the healthyr package to prevent others from running into the same problem?